src/main/protobuf/soc/gameState.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/board.proto";
+import "soc/inventory.proto";
+import "soc/playerState.proto";
+
+message GameState {
+    required Board board = 1;
+    repeated PlayerState playerStates = 2;
+    required int32 developmentCardsInDeck = 3;
+    required int32 currentTurn = 4;
+    required bool canRollDice = 5;
+    required bool canPlayCard = 6;
+    required ResourceSet bank = 7;
```

Comment:
I'm not sure I love `bank`.... `publicResources` publicResourceInventory` `resourceInventory` maybe `resourceBank` ? I don't love any of the other suggestions either, but I think it would be nice to get the word "Resource" in there.

Reply:
ok I guess `resource_bank` since I think the game refers to those cards as the bank

Reply:
Okay, I like resource bank, no that I know that the game refers to the as `bank`!